### ISSUE TITLE
Regarding use of regions and caption_rois

### ISSUE THREAD

**Issue by sandeep-ipk** [yikang-li/FactorizableNet#28]
Hey @yikang-li ,

Can you provide the model which makes use of external knowledge such as region captions, which is described in your paper? The current model (factorizable_network_v4.py) cannot incorporate the region captions and in the RPN_FN.yaml file, the configurations for objects are given but the configurations for regions are not given. Without the use of these external captions, the evaluation of the pretrained MSDN model is not giving the reported results in the README file.

**Comment by yikang-li** [comment]
Hi we didn't use the external captions, like caption regions, during the training in this job. The provided pre-trained model is trained using warm-up training. We trained the model for several epochs with learning rate decay, then reset the learning rate and train again. It will help to get a better result. I have provided the information in the last block of the "Training" section in README.
